cranelift/codegen/Cargo.toml

```diff
@@ -33,7 +34,7 @@ byteorder = { version = "1.3.2", default-features = false }
 cranelift-codegen-meta = { path = "meta", version = "0.62.0" }
 
 [features]
-default = ["std", "unwind"]
+default = ["std", "unwind", "all-arch"]
```

Comment:
`all-arch` was deliberately removed from the default in https://github.com/bytecodealliance/cranelift/pull/853. Is there a new need for it to be enabled here, or is this something that could be enabled by downstream users of cranelift?

Reply:
Sorry, that was added for development convenience only; reverted.